tmol/score/omega/potentials/params.hh

```diff
@@ -43,4 +58,21 @@ struct enable_tensor_view<score::omega::potentials::OmegaParameters<Real>> {
   typedef typename enable_tensor_view<Real>::PrimitiveType PrimitiveType;
 };
 
+template <typename Real>
+struct enable_tensor_view<score::omega::potentials::OmegaGlobalParams<Real>> {
+  static const bool enabled = enable_tensor_view<Real>::enabled;
+  static const at::ScalarType scalar_type() {
+    return enable_tensor_view<Real>::scalar_type();
+  }
+
+  static const int nconsumed_dims = 1;
+  static const int consumed_dims(int i) {
+    return (i == 0) ? sizeof(score::omega::potentials::OmegaGlobalParams<Real>)
+                          / sizeof(Real)
+                    : 0;
+  }
+
+  typedef typename enable_tensor_view<Real>::PrimitiveType PrimitiveType;
+};
```

Comment:
I just copy/pasted this from LJLK, and don't really know what's going on here. Are there any specific considerations? Is this code that can be factored out somehow?